README.md

```diff
@@ -0,0 +1,116 @@
+  <strong>
+    <h1 align="center" >create-express-ts-api</h3>
+<p align="center">
+    <strong>
+      A starter to kit for developing  REST API in typescript and express.
```

Comment:
Somewhere after this add a link hyperlink to the npm package along with the npm badge.
Also, it would be good itfwe will use tech stack badges here too

Reply:
Done